README.md

```diff
@@ -101,7 +103,6 @@ tape ansible deploy -l staging
 ## Testing
```

Comment:
I wonder if this whole testing and development section could be moved to a CONTRIBUTING.md